### DISCUSSION TITLE
Terms and Conditions link not visible in Workload Page

### DISCUSSION THREAD

**Discussion by SanchanaMohankumar** [microsoft/Microsoft-Fabric-workload-development-sample#129] [Q&A]
Hi Team, 
As seen in Workload Image 1, the "Terms and Conditions" link is missing, despite being included in our Product.json file. In contrast, Image 2, which shows an older version, displays the "Terms and Conditions" link correctly. Additionally, in Image 3, the Azure Marketplace offer page link is not visible on certain screen sizes. In our scenario, with more description data, the Azure Marketplace link becomes hidden. Could you please provide your input on these issues?
Image 1: New Workload Page
<img width="959" alt="image" src="https://github.com/user-attachments/assets/e1c6cf18-9adb-4bf9-aba3-56515f5c72b4">
Image 2: Old Workload Page
![Termsandcondition](https://github.com/user-attachments/assets/d20df795-dd34-4ceb-a62f-69197b3cfdc6)
Image 3: Workload Page screen size issue 
<img width="959" alt="image" src="https://github.com/user-attachments/assets/88a82d4b-fd95-42bb-9f45-ba6c4db14ac1">

Thank you

**Comment by michal-mano** [comment]
Hello,
Yes, there appears to be a bug with the newer design of the workload hub page and the terms and conditions are no longer visible, thank you for bringing this to our attention!
We will address this, and your additional comment, as soon as possible. You do not need to make changes to the structure of the Product.json.
